.github/CONTRIBUTING.md

```diff
@@ -1,7 +1,7 @@
 # How to contribute
 
 We welcome contributions from external contributors, and this document
-describes how to merge code changes into this project_name. 
+describes how to merge code changes into QCElemental. 
```

Comment:
Maybe add the below here:

For queries beyond GitHub mechanics, please ask in an issue or follow the "chat on slack" on [README.md](https://github.com/MolSSI/QCElemental/blob/master/README.md) to the QCElemental user group.